torch/_meta_registrations.py

```diff
@@ -434,15 +432,6 @@ def meta__linalg_eigh(
     shape.pop()
     vals = A.new_empty(shape, dtype=toRealValueType(A.dtype))
 
-    if eigenvalues is not None and eigenvectors is not None:
-        assert isinstance(eigenvalues, TensorLike)
-        assert isinstance(eigenvectors, TensorLike)
-        eigenvalues = _maybe_resize_out(eigenvalues, vals.shape)
-        eigenvectors = _maybe_resize_out(eigenvectors, vecs.shape)
-        _safe_copy_out(copy_from=vals, copy_to=eigenvalues)  # type: ignore[arg-type]
-        _safe_copy_out(copy_from=vecs, copy_to=eigenvectors)  # type: ignore[arg-type]
-        return eigenvalues, eigenvectors
```

Comment:
Was needed due to `test_dispatch_meta_outplace_linalg_eigvalsh_cpu_float64` failing, see: https://github.com/pytorch/pytorch/pull/100964#discussion_r1189208522